src/components/TabularReport/ReportTable.js

```diff
@@ -0,0 +1,25 @@
+/* React */
+import React, { Fragment } from 'react'
+import PropTypes from 'prop-types'
+
+const Header = (header, index) => <th key={index}>{header}</th>
```

Comment:
just using a number as a key is dangerous in this case if we have the html report below the form. After requesting a new report and feeding the changes to this component, the keys won't change and it might produce render bugs.

I'm not so sure what `header`, `row` and `cell` are going to be. If it's just a string, you might want to use that as the key, it it's an object with an id, the id would be perfect, like `key={index + cell}` or `key={index + cell.is}`.

Reply:
I'll change this to be more compliant, but I don't it matters in this case. The table follows this render cycle:
1. Render nothing
2. Render loader
3. Render table
4. Render loader
5. Render table

At no point in time are we replacing rows. Keys are important when children are dynamic. Our children are not.

https://stackoverflow.com/questions/42801343/what-is-the-significance-of-keys-in-reactjs/42801409
> TL;DR Use unique and constant keys when rendering dynamic children, or expect strange things to happen.